extensions/amp-ventuno-player/0.1/amp-ventuno-player.js

```diff
@@ -0,0 +1,339 @@
+        'The data-slotid attribute is required for <amp-ventuno-player> %s',
+        this.element);
+  }
+
+  /** @return {string} */
+  getVideoIframeSrc_() {
+    const pubid = encodeURIComponent(this.pubid_ || ''),
+        slotid = encodeURIComponent(this.slotid_ || ''),
+        pType = this.playerType_ || '',
+        optParams = {};
+
+    if (this.videoIframeSrc_) {
+      return this.videoIframeSrc_;
+    }
+    // As of now, only the 'ep' player type will be supported
+    user().assert(this.playerType_ === 'ep',
+        'Only Editorial Player is supported');
+
+    dev().assert(this.pubid_);
+    dev().assert(this.slotid_);
+
+    let src = `https://venwebsecure.ventunotech.com/embed/embedPlayer.html?pFrom=amp&pType=${pType}&pubKey=${pubid}&slot=${slotid}`;
+
+    if (this.title_) {
+      optParams['pTitle'] = this.title_;
+    }
+
+    if (this.url_) {
+      optParams['pUrl'] = this.url_;
+    }
+
+    if (this.meta_) {
+      optParams['pMeta'] = this.meta_;
+    }
+
+    src = addParamsToUrl(src, optParams);
+
+    return this.videoIframeSrc_ = src;
+  }
+
+  /**
+   * Sends a command to the player through postMessage.
+   * @param {string} command
+   * @param {Array=} opt_args
+   * @private
+   */
+  sendCommand_(command) {
+    this.playerReadyPromise_.then(() => {
+      if (this.iframe_ && this.iframe_.contentWindow) {
+        const message = 'vtn' + command;
+        this.iframe_.contentWindow./*OK*/postMessage({
+          command: message,
+          from: 'amp',
+        }, '*');
+      }
+    });
+  }
+
+  // VideoInterface Implementation. See ../src/video-interface.VideoInterface
+  /**
+   * @override
+   */
+  supportsPlatform() {
+    return true;
+  }
+
+  /** @override */
+  isInteractive() {
+    return true;
+  }
+
+  /**
+   * @override
+   */
+  play(unusedIsAutoplay) {
+    this.sendCommand_('Play');
+  }
+
+  /**
+   * @override
+   */
+  pause() {
+    this.sendCommand_('Pause');
+  }
+
+  /**
+   * @override
+   */
+  mute() {
+    this.sendCommand_('Mute');
+  }
+
+  /**
+   * @override
+   */
+  unmute() {
+    this.sendCommand_('UnMute');
+  }
+
+  /**
+   * @override
+   */
+  showControls() {
+    // Not supported.
+  }
+
+  /**
+   * @override
+   */
+  hideControls() {
+    // Not supported.
+  }
+
+  /**
+   * @override
+   */
+  fullscreenEnter() {
+    if (!this.iframe_) {
+      return;
+    }
+    fullscreenEnter(dev().assertElement(this.iframe_));
+  }
+
+  /**
+   * @override
+   */
+  fullscreenExit() {
+    if (!this.iframe_) {
+      return;
+    }
+    fullscreenExit(dev().assertElement(this.iframe_));
+  }
+
+  /** @override */
+  isFullscreen() {
+    if (!this.iframe_) {
+      return false;
+    }
+    return isFullscreenElement(dev().assertElement(this.iframe_));
+  }
+
+  /** @override */
+  getMetadata() {
+    // Not implemented
+  }
+
+  /** @override */
+  preimplementsMediaSessionAPI() {
+    // Youtube already updates the Media Session so no need for the video
```

Comment:
Does your player actually implement the Media Session API? If not, set this to false.